src/main/webapp/app/modeling-editor/modeling-editor.component.ts

```diff
@@ -42,6 +43,10 @@ export class ModelingEditorComponent implements OnInit, OnDestroy, ComponentCanD
     positions: {};
```

Comment:
There are still attributes with type "any" --> Please get rid of any in all code and specify the type properly!